### ISSUE TITLE
Document RateMap etc

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tskit#2679]
At the moment the documentation is in msprime under its own section within "utilities": [Rate Maps](https://tskit.dev/msprime/docs/stable/rate_maps.html). Since it is purely a python thing, I wonder if in `tskit` it should simply live in the Python API documentation. We already have a [utility functions](https://tskit.dev/tskit/docs/stable/python-api.html#utility-functions) section: we could simply make it "Utility classes and functions" and put some of the explanatory stuff there? Some might want to stay in the msprime docs, I guess.

**Comment by jeromekelleher** [comment]
Do we want to make this a blocker for 0.5.4?

**Comment by hyanwong** [comment]
No, I don't think that's necessary if we want to get something out. The msprime version will carry on working and is documented there. It will take some thought to get the docs right, so I think it can just be undocumented for the time being.

**Comment by jeromekelleher** [comment]
I think there are good reasons for not documented it yet actually - let's see how the process of cutting it out of msprime goes first and whether we need to make any minor API tweaks first.

**Comment by hyanwong** [comment]
Yes, I agree.

**Comment by hyanwong** [comment]
Just looking at this again, in the light of wanting to be able to export a HapMap-style recombination rate. I wonder if we want to define `class RecombinationMap(RateMap):`, and then have `read_hapmap` (and maybe `write_hapmap`) as methods of the subclass?
